extension/src/experiments/index.ts

```diff
@@ -466,13 +465,6 @@ export class Experiments extends BaseRepository<TableData> {
     return this.experiments.getExperiments()
   }
 
-  public getExperimentDisplayName(experimentId: string) {
```

Comment:
[F] id is now the same as this so we can just pass the id around everywhere and remove this logic. This leads to a lot of code getting deleted from `WorkspaceExperiments`.